### PR TITLE
[TASK] Move `fluid javascript variables` to FrontendEditing object.

### PR DESCRIPTION
**Pull request by phathoang**
Need this one to be able to use translated labels and other stuff that was in fluid ..

### CODE REVIEW

File: Classes/Hook/ContentPostProc.php [hunk 1]
```diff
@@ -67,11 +68,6 @@ public function main(array $params, \TYPO3\CMS\Frontend\Controller\TypoScriptFro
 
                 $output = $this->loadResources();
 
-                $iframeUrl = GeneralUtility::getIndpEnv('TYPO3_SITE_URL') .
-                    'index.php?id=' . $this->typoScriptFrontendController->id .
-                    '&frontend_editing=true'
-                ;
-
                 //\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance
```
Comment (MattiasNilsson) [inline Classes/Hook/ContentPostProc.php]: Remove unused line.

File: Classes/Hook/ContentPostProc.php [hunk 2]
```diff
@@ -117,6 +110,50 @@ public function main(array $params, \TYPO3\CMS\Frontend\Controller\TypoScriptFro
     }
 
     /**
+     * Get all kind of javascript needed in FE
+     *
+     * @return array
+     */
+    protected function getJavascriptForFrontendEditing()
+    {
+        $jsArray = [
```
Comment (MattiasNilsson) [inline Classes/Hook/ContentPostProc.php]: Use explicit names javascriptArray

File: Classes/Hook/ContentPostProc.php [hunk 3]
```diff
@@ -117,6 +110,50 @@ public function main(array $params, \TYPO3\CMS\Frontend\Controller\TypoScriptFro
...
+    protected function getLocalizedFrontendLabels()
+    {
+        $languageFactory = GeneralUtility::makeInstance('TYPO3\\CMS\\Core\\Localization\\LocalizationFactory');
+        $parsedLocallang = $languageFactory->getParsedData(
+            'EXT:frontend_editing/Resources/Private/Language/locallang.xlf',
+            'default'
+        );
+        $localizedLabels = [];
+        foreach (array_keys($parsedLocallang['default']) as $key) {
+            if (strpos($key, 'notifications.') === 0 ||
```
Comment (MattiasNilsson) [inline Classes/Hook/ContentPostProc.php]: Why the notifcations. ? If we add a new section to the locallang we also need to add it here?

Comment (phathoang) [inline Classes/Hook/ContentPostProc.php]: Yeh, or we need to load all labels into a huge array. Or name all fe-labels 'fe.xxxxx...', or something with typoscript.

Comment (phathoang) [inline Classes/Hook/ContentPostProc.php]: 063ddfe

File: Classes/Hook/ContentPostProc.php (hunk 2 above)
Comment (phathoang) [inline Classes/Hook/ContentPostProc.php]: 063ddfe

**Pull request merged**